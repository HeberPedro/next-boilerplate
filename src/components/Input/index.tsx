import { InputHTMLAttributes } from 'react'
import * as S from './styles'

export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
  children?: React.ReactNode
}

const Input = ({ children }: InputProps) => <S.Wrapper>{children}</S.Wrapper>

export default Input
