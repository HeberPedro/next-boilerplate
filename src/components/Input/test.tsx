import { render } from '@testing-library/react'

import Input from '.'

describe('[COMPONENT]: Input', () => {
  it('It should render input', () => {
    const mockText = 'Test'

    const { container } = render(<Input name={mockText} />)

    expect(container).toMatchSnapshot()
  })
})
