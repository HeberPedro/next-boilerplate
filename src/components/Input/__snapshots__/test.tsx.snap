// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[COMPONENT]: Input It should render input 1`] = `
<div>
  <input
    class=""
  />
</div>
`;
